tsconfig-build-options.json

```diff
@@ -6,7 +6,9 @@
         "declarationMap": true
     },
     "exclude": [
+        // Implies there's a types-index.d.ts already which shouldn't be overwritten
```

Comment:
Oh comments are supported in these `.json` files!

Looks like for [local resolution, `.jsonc` seem to be supported](https://redirect.github.com/microsoft/TypeScript/issues/43121). Wondering if we should switch to that?

Reply:
Good question but I don't think it's worth it because we couldn't do it for all the tsconfig